packages/generator/src/complex-type/namespace.ts

```diff
@@ -33,7 +33,17 @@ function factoryFunction(
 }
 
 function getJsonType(complexType: VdmComplexType): string {
-  return '{ [keys: string]: FieldType }';
+  const complexTypes = new Set(
+    complexType.properties
+      .filter(prop => prop.isComplex)
+      .map(prop => prop.jsType)
+      .sort()
```

Comment:
Why is this sort needed. It is anyways put into a set... If this is really necessary, I would rather expect it below when the array is constructed. BTW, there is a `unique` function in the `@sap-cloud-sdk/util` package.